RecipeApp/Modules/RecipeBook/FlowCoordinator/RecipeBookCoordinator.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  RecipeBookCoordinator.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-17.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import UIKit
+
+class RecipeBookCoordinator: Coordinator {
+    
+    var navigationController: UINavigationController
+
+   init(navigationController: UINavigationController) {
+       self.navigationController = navigationController
+   }
+
+    func start() {
+        let recipeBookViewModel = RecipeBookViewModel()
+        let recipeBookViewController = RecipeBookViewController.create(viewModel: recipeBookViewModel)
+        recipeBookViewController.coordinator = self
+        navigationController.pushViewController(recipeBookViewController, animated: true)
+        recipeBookViewController.tabBarItem.title = "Favorites"
+        recipeBookViewController.tabBarItem.image = UIImage(named: "Favorite")
+        
+        recipeBookViewModel.completion = { [weak self] meal in
+            self?.viewRecipeDetails(meal: meal)
+        }
+    }
+    
+    func viewRecipeDetails(meal: Meal) {
+        let recipeDetailsViewController = RecipeDetailsCoordinator(navigationController: navigationController)
```

Comment:
You're creating a coordinator but calling it a viewController. Why aren't you using a viewModel? If we are using MVVM then you should be consistent. Any reason why you chose to set a property on the coordinator rather than just add another parameter to the initializer?